src/database.ts

```diff
@@ -460,6 +605,8 @@ class Database extends ServiceObject {
       }
     });
   }
+  delete(): Promise<[r.Response]>;
```

Comment:
This should have its own type and we should be using `IEmpty` in this case.

Reply:
Or is this another case of conflicts with the base class?

Reply:
yes conflict with base class...